config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
@josegonzalez 

I'm using CakePHP 3.1.8.

I figured it out what's going here.

If I only set `DATABASE_URL` (without `DATABASE_TEST_URL`), CakePHP failed to create a connection with the empty url for the test connection. Somehow it can not connect to the DB.

If I set both `DATABASE_URL` and `DATABASE_TEST_URL`, it works just fine.

Is it expected?